source/production/F0.Minesweeper.Logic.Abstractions/MinefieldOptions.cs

```diff
@@ -1,16 +1,14 @@
-using System;
-
 namespace F0.Minesweeper.Logic.Abstractions
 {
-	public sealed record MinefieldOptions(uint Width, uint Height, uint MineCount)
+	public sealed record MinefieldOptions(uint Width, uint Height, uint MineCount, MinefieldFirstUncoverBehaviour GenerationOption)
 	{
 		public static MinefieldOptions Easy
-			=> new MinefieldOptions(10, 8, 10);
+			=> new(10, 8, 10, MinefieldFirstUncoverBehaviour.WithoutAdjacentMines);
```

Comment:
I really like the _target-typed new expressions_ here

Reply:
Yep. So simple and beautiful 😍